### DISCUSSION TITLE
Working with each tenant tables

### DISCUSSION THREAD

**Discussion by shankarGdev** [spatie/laravel-multitenancy#555] [Q&A]
I'm using multiple databases option.

I'm building admin panel for managing all tenants, so how to access details for each tenants stored in their database and perform update operation in it.

Can anyone guide me please.

**Comment by masterix21** [comment]
What kind of update operation? Can't be accomplished by a job?

**Reply by shankarGdev** [reply]
We have `features` table in each tenant database, we show then (admin tenant features page) and update the value based on tenant.

We have tenant list and click on features will open that new page, where we show that and do the update.

**Comment by shankarGdev** [comment]
Tried following code

```
$tenant = Tenant::findOrFail($id);
$tenant->makeCurrent();

// Fetch tenant db
$features = Feature::all();
```

above code works with session drive `file` & `cookie`.

But not working with `database` session driver.
gettign error `Call to a member function prepare() on null`
Error Log: https://flareapp.io/share/NPLEAOK7

**Reply by masterix21** [reply]
Change your default connection to landlord.

**Reply by shankarGdev** [reply]
When I try with `DB_CONNECTION=landlord`, getting following error 

```
`SwitchTenantDatabaseTask` fails because `multitenancy.tenant_database_connection_name` is `null` or equals to `multitenancy.landlord_database_connection_name`.
```

also in my `.env` I have 

```
DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=landlord_db
DB_USERNAME=root
DB_PASSWORD=password
```

application will connect to landlord by default.
